src/PKSim.Infrastructure/Services/SimulationTransferLoader.cs

```diff
@@ -27,7 +27,7 @@ public SimulationTransfer Load(string pkmlFileFullPath)
       {
          var project = _projectRetriever.CurrentProject;
 
-         //use new ObjectBaseRepository here as the resulting simulation will be registered later on when added to the project
+         //use new WithIdRepository here as the resulting simulation will be registered later on when added to the project
          var simulationTransfer = _simulationPersister.Load(pkmlFileFullPath, new WithIdRepository());
```

Comment:
Interestingly, we do not load obs data when loading from MoBi (this is not really a use case that is done often). So I am not going to tackle this for now